fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -689,6 +690,8 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     private long spillOperatorMinBytes = 1024L * 1024 * 10;
     @VarAttr(name = SPILL_OPERATOR_MAX_BYTES, flag = VariableMgr.INVISIBLE)
     private long spillOperatorMaxBytes = 1024L * 1024 * 1000;
+    @VarAttr(name = SPILL_ENCODE_LEVEL)
+    private int spillEncodeLevel = 7;
```

Comment:
how to set the encode level value?

Reply:
maybe we could use encode-level

Reply:
> how to set the encode level value?

similar to the descriptions in https://github.com/StarRocks/starrocks/pull/11251, I have add them into code comments

Reply:
> maybe we could use encode-level

we may use different encode-level in different scenario, should give them different prefix in session var